settings/settings.go

```diff
@@ -32,7 +32,7 @@ var (
 	settings Config
 
 	// Version 2.0.0
-	newAirflowVersion uint64 = 2
+	NewAirflowVersion uint64 = 2
```

Comment:
New - is slightly confuse me, cause it's de facto constructor of objects in golang.
can we rename to something more readable rather than just `NewAirflowVersion`?
`AirflowVersionSecond`?

Reply:
Yeah, I didn't know what we should call it because it's a special case.  `AirflowVersionSecond` or `AirflowVersionTwo`?

Reply:
yep but i. prefer to name explain value, since it's airflow 2.0.0 specific.
`AirflowVersionTwo` +1

Reply:
Updated to `AirflowVersionTwo`